### PR TITLE
feature: added sprint planning workflow

### PR DESCRIPTION
**Pull request by theantichris**

### CODE REVIEW

File: docs/sprint-planning.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Assign Points to any Stories that are unestimated
+1. Go to the "Everything" Space
+1. Uncheck the "No Iteration" Filter
+1. Go through each Column starting with "Acceptance", discuss the status of each Story, and take action if needed
+1. If any Story in the "Ready For Development" Column will not be started during the current Iteration move them to the next Iteration
+1. Go to the "Reports" page
+1. Look at the last 3 months to determine the team's average velocity
+1. Go back to the "Everything" Space on the "Stories" page
+1. Review and assign WordPress Legacy work that needs to be prioritized for the next Iteration
+1. Go to the "Milestone" page for each team
```
Comment (jmosley5) [inline docs/sprint-planning.md:20]: ❓ This is a little bit confusing to me. Would we look at each epic within the milestone?

**Review by jmosley5 — APPROVED** [review]
👏🏼 Thank you for putting this together! This is something I've been confused by even though we've done this as a team many times.

File: docs/sprint-planning.md (hunk 1 above)
Comment (theantichris) [reply docs/sprint-planning.md:20]: You shouldn't have to go into each epic. The team for that project should know what they need to work on next and assign that.

**Review by ryexley — DISMISSED** [review]

**Review by racheljay — DISMISSED** [review]

**Review by ishmaelavila — DISMISSED** [review]

File: docs/sprint-planning.md (hunk 1 above)
Comment (rlmead) [inline docs/sprint-planning.md:20]: by "each team" do you mean Software & Product?

File: docs/sprint-planning.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+1. Go to the "Stories" page in Shortcut
+1. Go to the "Needs Iteration" Space
+1. Assign the current Iteration to any Story that does not have an Interation and is in progress
+1. Go to the "Needs Points" Space
+1. Assign Points to any Stories that are unestimated
+1. Go to the "Everything" Space
+1. Uncheck the "No Iteration" Filter
+1. Go through each Column starting with "Acceptance", discuss the status of each Story, and take action if needed
```
Comment (rlmead) [inline docs/sprint-planning.md:14]: does this mean "go through each column from right-to-left"?

**Review by scott-daily — DISMISSED** [review]

File: docs/sprint-planning.md (hunk 2 above)
Comment (theantichris) [reply docs/sprint-planning.md:14]: Yes, do you feel that needs clariication?

File: docs/sprint-planning.md (hunk 1 above)
Comment (theantichris) [reply docs/sprint-planning.md:20]: No, I mean each software team. Right now "Product Activation" and "Marketplace".

Comment (theantichris) [reply docs/sprint-planning.md:20]: Let me know how I can add clarification if needed.

Comment (rlmead) [reply docs/sprint-planning.md:20]: ah ok, then maybe something like
> Go to the page for each Milestone that's in progress

?

File: docs/sprint-planning.md (hunk 2 above)
Comment (rlmead) [reply docs/sprint-planning.md:14]: i think so, maybe
> Go through each Column from right to left (starting with "Acceptance") ...

**Review by Thomas-Jean — APPROVED** [review]

**Review by scott-daily — APPROVED** [review]

**Review by jmosley5 — APPROVED** [review]

File: docs/sprint-planning.md (hunk 2 above)
Comment (rlmead) [inline docs/sprint-planning.md:14]:
```suggestion
1. Go through each Column from right to left starting with "Acceptance", discuss the status of each Story, and take action if needed
```

File: docs/sprint-planning.md (hunk 1 above)
Comment (rlmead) [inline docs/sprint-planning.md]:
```suggestion
1. Go to the "Milestone" page for each Milestone that's in progress
```

**Review by rlmead — CHANGES_REQUESTED** [review]

**Review by ishmaelavila — DISMISSED** [review]

**Review by bawilc01 — APPROVED** [review]

**Review by racheljay — APPROVED** [review]

**Review by ryexley — DISMISSED** [review]

**Review by jmosley5 — APPROVED** [review]

**Review by ryexley — APPROVED** [review]

**Review by racheljay — APPROVED** [review]

**Review by rlmead — APPROVED** [review]

**Pull request merged**